Packs/ProofpointTAP/Integrations/ProofpointTAP_v2/ProofpointTAP_v2.py

```diff
@@ -444,7 +616,589 @@ def fetch_incidents(client, last_run, first_fetch_time, event_type_filter, threa
+    intervals = []
+    if current_time - time_range > timedelta(
+            days=7):  # The maximum time range of Proofpoint TAP API requests is 7 days minus one minute.
+        time_range += timedelta(minutes=1)
+
+    if is_days_interval:
+        while current_time - time_range > timedelta(days=1):
+            start = time_range.strftime(DATE_FORMAT)
+            time_range += timedelta(days=1)
+            intervals.append(f'{start}/{time_range.strftime(DATE_FORMAT)}')
+
+    if is_hours_interval:
+        while current_time - time_range > timedelta(hours=1):
+            start = time_range.strftime(DATE_FORMAT)
+            time_range += timedelta(hours=1)
+            intervals.append(f'{start}/{time_range.strftime(DATE_FORMAT)}')
+
+    return intervals
+
+
+def get_clicks_command(client: Client, is_blocked: bool, interval: str = None, threat_status: str = None,
+                       time_range: str = None) -> CommandResults:
+    """
+    Retrieves clicks on malicious URLs in the specified time period. Clicks can either be blocked or permitted.
+    Args:
+        client (Client): ProofpointTAP API client.
+        is_blocked (bool): Indicates the clicks type.
+        interval (str): ISO8601-formatted interval date. The minimum interval is thirty seconds. The maximum interval is one hour.
+        threat_status (str): The status of the threat. Can be: active, cleared or falsePositive.
+        time_range (str): Time range, for example: 1 week, 2 days, 3 hours etc.
+    Returns:
+        CommandResults: raw response, outputs, and readable outputs.
+    """
+    clicks_type = 'blocked' if is_blocked else 'permitted'
+    if not (interval or time_range):
+        raise Exception('Must provide interval or time_range.')
+    if interval and time_range:
+        raise Exception('Must provide only one of the arguments interval or time_range.')
+    if time_range and dateparser.parse("7 days") > dateparser.parse(time_range):
+        raise Exception('The maximum time range is 7 days')
+    if time_range and dateparser.parse("30 seconds") < dateparser.parse(time_range):
+        raise Exception('The minimum time range is thirty seconds.')
+
+    if time_range and dateparser.parse("1 hour") < dateparser.parse(time_range):
+        end = datetime.utcnow().strftime(DATE_FORMAT)
+        start = dateparser.parse(time_range).strftime(DATE_FORMAT)
+        intervals = [f'{start}/{end}']
+    else:
+        intervals = handle_interval(dateparser.parse(time_range)) if time_range else [interval]  # type: ignore
+
+    outputs = []
+    raw_responses = []
+    request_error = []
+    for interval_string in intervals:
+        try:
+            raw_response = client.get_clicks(clicks_type, interval_string, threat_status)
+        except Exception:
+            request_error.append(
+                {'interval': interval_string,
+                 "message": f'Error retrieving data from {interval_string}. Please validate the interval format'})
```

Comment:
I agree with you, but the main reason I referred to such a case is because of the 'list-campaigns' command.
When there is an API request with an interval where there are no campaigns - an error is thrown. The error is thrown from the implementation of the 'http_request' function that existed in the previous integration (the reason is that the API returns 404 status - 'Not found', and therefore the error is thrown).
Would you suggest keep this reference only in the 'list_campaigns_command' function?